### PR TITLE
PkgBase.live: add an un-update

### PR DESCRIPTION
**Pull request by igalic**
PkgBase.live is dead (for the time being)

### CODE REVIEW

**Review by lsalvadore — APPROVED** [review]

**Comment by lsalvadore** [comment]
I have added my doc mentors @sergio-carlavilla and @dbaio to the review to get their mentor approval before merging.

File: website/content/en/status/report-2023-01-2023-03/pkgbase.live.adoc
Comment (pauamma) [inline website/content/en/status/report-2023-01-2023-03/pkgbase.live.adoc:4]: I'd suggest "defunct" (or "dead").

Comment (igalic) [reply website/content/en/status/report-2023-01-2023-03/pkgbase.live.adoc:4]: my spellchecker didn't complain, but I'll change it.

**Comment by dbaio** [comment]
> I have added my doc mentors @sergio-carlavilla and @dbaio to the review to get their mentor approval before merging.

One improvement that could be made here is to hide the defunct link. While it explicitly indicates that it is not working, it may still tempt people to try it out as I did. =)

```
Links: +
Website (defunkt or defunct) URL: \https://alpha.pkgbase.live +
link:https://codeberg.org/pkgbase/website[Website source] URL: link:https://codeberg.org/pkgbase/website[]
```

@lsalvadore  whatever you decide, you have my approval, with or without it.

Thank you all.

**Comment by lsalvadore** [comment]
I agree with @dbaio: link to a defunct website does not look right. But I think we can do better than removing the link: we can link to an archived version of the website, so that although people cannot access it anymore, they can still see how it was and thus be tempted to copy it or provide some hardware.
 For example, I have found this link: http://web.archive.org/web/20221220222828/https://alpha.pkgbase.live/

Anyway @igalic, I would like to leave to you the decision about what to do with that link. Once you have decided I will merge this pull request.

**Comment by lsalvadore** [comment]
Merged with commit https://github.com/freebsd/freebsd-doc/commit/8a25677c39aa9eda7c3fedc6ad6b4acef061ada7, thanks.

@igalic: I have noticed that you are missing from the additional contributors list. I have opened review https://reviews.freebsd.org/D39362 to fix that.

**Pull request closed without merging**
